src/charm.py

```diff
@@ -196,7 +363,7 @@ def _pebble_layer(self) -> Layer:
                     self._service_name: {
                         "override": "replace",
                         "startup": "enabled",
-                        "command": f"/free5gc/ausf/ausf --ausfcfg {CONFIG_DIR}/{CONFIG_FILE_NAME}",  # noqa: E501
+                        "command": f"/bin/ausf --ausfcfg {CONFIG_DIR}/{CONFIG_FILE_NAME}",  # noqa: E501
```

Comment:
The change of the command and some other changes I spotted here (like the image name in the README) are already merged, it might be useful to rebase before merging this pr :)

Reply:
Oops, should be fixed now